build.gradle

```diff
@@ -173,7 +173,7 @@ sonarqube {
     property "sonar.projectKey", "uk.gov.hmcts.reform:cpo-case-payment-orders-api"
     property "sonar.coverage.jacoco.xmlReportPaths", "${jacocoTestReport.reports.xml.destination.path}"
     property "sonar.coverage.exclusions", "**/security/SecurityUtils.java,**/CasePaymentOrdersServiceImpl.java,**/CasePaymentOrderQueryFilter.java,**/CasePaymentOrderEntity.java,**/CaseIdsValidator.java"
-    property "sonar.exclusions","**/config/SecurityConfiguration.java"
+    property "sonar.exclusions","**/config/SecurityConfiguration.java,**/config/HibernateEventListenerRegistryConfiguration.java"
```

Comment:
Is it better to suppress specific items in the class (with a comment to justify if controversial) rather than to exclude all checks in class?